frc/1. Basics/5. Motors.md

```diff
@@ -0,0 +1,118 @@
+# Motors
+
+## Terminology
+
+An actuator is defined as "a component of a machine that is responsible for moving and controlling a mechanism or system." For our purposes, we need only consider *motors* and *pneumatics*. 
+
+Our robot is essentially entirely made up of motors. All moving components are controlled by motors or pneumatics, and our job as programmers is to figure out how to convert drive team input into the robot actually doing its job. For instance, programming will make sure that certain motors move when drive team pushes buttons or joysticks on the controller.
+
+First, some clarification:
+
+- *Motor* - the actual physical part that spins
+- *Motor Controller* - facilitator that takes in and sends signal to control the motor accordingly
+
+Motors are widely used throughout most, if not all of our robot's subsystems, e.g. on the drivetrain. At the time of writing, 1257 mainly used NEO Brushless and NEO 550 motors, by REV Robotics. 
+
+<img src="img/SPARKMAX.png" width="330" height=""> <img src="img/NEOvs550.png" width="330"> 
+
+To control these motors and actually manipulate a robot mechanism, we interface with the corresponding motor controllers--this is what we do in code. The SPARK MAX controller (also by REV Robotics) is used with NEO-type motors. 
+
+## Setup
+
+Here, we'll write in the iterative programming paradigm for brevity. Examples for command-based programming will follow in the next few files.
```

Comment:
try not to talk about iterative rn, i want to avoid confusion that we faced this year. just put the motor code in by itself (not in a function) and tell them that we will go over fitting this into our subsystem later